charts/access/slack/values.yaml

```diff
@@ -23,6 +23,18 @@ log:
 
 secretVolumeName: "password-file"
 
+# Kubernetes annotations to apply
+# https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+annotations:
+  # Annotations for the ConfigMap
+  config: {}
```

Comment:
can you please rename this to confirmap

Reply:
I don't think that should be done.

I was explicitly following the existing pattern for the [teleport-kube-agent](https://goteleport.com/docs/reference/helm-reference/teleport-kube-agent/#annotationsconfig) and [teleport-cluster](https://goteleport.com/docs/reference/helm-reference/teleport-cluster/#annotationsconfig) helm charts, both of which use `config` rather than `configmap`.

If I'm going to change this to `configmap`, both of those (and any other charts using the same pattern which I'm not aware of), should also be altered. I think that's probably outside the scope of this change.

Reply:
Oh, you are correct.
I completely missed that we name everything as config :cry: